createuserpkg

```diff
@@ -98,7 +98,9 @@ def main():
         exit(-1)
     filename = arguments[0]
 
-    if options.password:
+    if options.password == 'none':
```

Comment:
The cynic in me knows that someone is going to file an issue because now they can't set the account password to "none" via the options...